lib/src/menu.dart

```diff
@@ -315,53 +315,52 @@ class TextMenuAction extends MenuAction {
   final IconData? trailingIcon;
 
   /// Style for the text.
-  final TextStyle style;
+  final TextStyle? style;
 
   /// Offset for the text from the left.
   final double leftTextOffset;
 
   @override
-  Widget buildAction(BuildContext context) => InkWell(
-        onTap: () => onTap.call(context),
-        child: Padding(
-          padding: const EdgeInsets.symmetric(vertical: 20),
-          child: SizedBox(
-            child: Row(
-              children: [
-                Expanded(
-                  flex: 1,
-                  child: (icon != null)
-                      ? Icon(
-                          icon,
-                          size: (style.fontSize ?? 16) * 1.5,
-                          color: style.color,
-                        )
-                      : SizedBox(
-                          width: leftTextOffset,
-                        ),
-                ),
-                Expanded(
-                  flex: 3,
-                  child: Row(
-                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                    children: [
-                      Text(
-                        text,
-                        style: style,
-                      ),
-                      if (trailingIcon != null) ...[
-                        Icon(
-                          trailingIcon,
-                          size: style.fontSize ?? 16,
-                          color: style.color,
-                        ),
-                      ],
-                    ],
+  Widget buildAction(BuildContext context) {
+    var theme = Theme.of(context);
+    return InkWell(
+      onTap: () => onTap.call(context),
+      child: Padding(
+        padding: EdgeInsets.symmetric(vertical: 20, horizontal: leftTextOffset),
+        child: SizedBox(
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Row(
+                children: [
+                  if (icon != null) ...[
+                    Icon(
+                      icon,
```

Comment:
you could assign the textstyle to a variable on line 325 and only do *1.5 on this line. That is more readable